core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/extensions/pgbouncer/extension.yml

```diff
@@ -1,38 +1,44 @@
 ---
 - name: Extensions | configure PgBouncer
-  when: groups['postgresql'][0] == inventory_hostname
+  when: groups.postgresql[0] == inventory_hostname
   block:
-    - name: Extensions | PgBouncer | Change pgbouncer configuration
-      lineinfile:
-        path: /etc/pgbouncer/pgbouncer.ini
-        insertafter: '\[databases\]'
-        line: postgres = host=127.0.0.1 port=5432 dbname=postgres
-        backup: true
-
-    - name: Extensions | PgBouncer | Change pool mode
-      lineinfile:
-        path: /etc/pgbouncer/pgbouncer.ini
-        regexp: '^pool_mode ='
-        line: "pool_mode = transaction"
+    - name: Extensions | PgBouncer | Ensure that systemd service is started
+      systemd:
+        name: pgbouncer
+        state: started
 
-    - name: Extensions | PgBouncer | Change pgbouncer users configuration
-      lineinfile:
-        path: /etc/pgbouncer/userlist.txt
-        line: '"postgres" "*"'
-        create: true
+    - name: Extensions | PgBouncer | Copy configuration files
+      vars:
+        _files:
```

Comment:
I think we shouldn't overwrite existing configuration if we re-run apply. I would stay with previous configuration and just replace particular lines.

Reply:
I don’t agree. It’s done the same as in all other places. IMO everything we support should be configured by epiphany, not manually.

If we need to have an ability to modify this file, it should be done by epiphany, not manually.

Also this was done following this comment - https://github.com/epiphany-platform/epiphany/pull/1845#pullrequestreview-531363344